src/main/resources/standardTestCases/dont_strip_line_prefix/dont_strip_line_prefix.wdl

```diff
@@ -0,0 +1,19 @@
+task prefix {
+  command {
+    echo hello \
+    | wc -l
+  }
+  output {
+    String out = read_string(stdout())
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+}
+
+workflow wf_prefix {
+  call prefix
+  output {
+     prefix.out
```

Comment:
Since this is a new file, I'd prefer a new style output block (e.g. `String prefix_out = prefix.out`)